Scripts/OverlayControllers/PowerUpPicker.cs

```diff
@@ -47,7 +47,6 @@ public void SetWinningSide(TeamEnum value)
   public override void _Ready()
   {
     this.AutoWire();
-    Reset();
```

Comment:
Remove this, didnt seem to cause any issues but Im not 100% if its needed or not

Reply:
This wont be a problem as long as the scene doesnt have any children